src/logger.ts

```diff
@@ -43,8 +45,11 @@ export function sendCrashResponse({
   // right before sending the response, to make sure that no concurrent
   // execution sends the response between the check and 'send' call below.
   if (res && !res.headersSent) {
-    res.set(FUNCTION_STATUS_HEADER_FIELD, 'crash');
-    res.send((err.message || err) + '');
+    res.set(FUNCTION_STATUS_HEADER_FIELD, statusHeader);
+    res.status(500);
+    if (process.env.NODE_ENV !== 'production') {
```

Comment:
In production environments, does this function code path send a response? Not sure why we need this if and what happens in production.

Reply:
Yeah good call. I will update.

Reply:
Fixed